package.json

```diff
@@ -82,7 +82,7 @@
     "@bbc/psammead-sitewide-links": "^4.0.10",
     "@bbc/psammead-story-promo": "^5.1.0",
     "@bbc/psammead-story-promo-list": "^4.0.5",
-    "@bbc/psammead-storybook-helpers": "^8.2.5",
+    "@bbc/psammead-storybook-helpers": "^8.2.6",
```

Comment:
Since this was bumped up, `CHANGELOG.md` should be updated in psammead too

Reply:
The CHANGELOG, package-lock.json and package.json are now updated.

Reply:
Removed updates to CHANGELOG, package-lock.json and package.json as since the version change in psammead-stoybook-helpers is minor, the changes in the package are already in use. This means psammead-stoybook-helpers doesn't need to be upgraded && this resolves [the issue highlighted](https://github.com/bbc/psammead/pull/3291#discussion_r397214154) above

Reply:
Thank you for the updates 👍 I can still see some changes in `package-lock.json` though, Jenkins should stop complaining after you remove them

Reply:
My statement above was a bit off. Since the minor change is still being noted and being installed, an update to the CHANGELOG and version numbers is still required.

Reply:
Sounds good. Can you make sure that `psammead-storybook-helpers` version is bumped up in `package.json` too, not just in `package-lock.json`?

Reply:
Bumped the version. Sorry for the back and forth.